flexbox/src/main/res/layout/activity_minheight_test.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.flexbox.FlexboxLayout android:id="@+id/flexbox_layout"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    app:flexDirection="column">
+
+    <TextView
+        android:id="@+id/text1"
+        android:layout_width="wrap_content"
+        android:layout_height="0dp"
+        android:text="1"
+        app:layout_flexGrow="1"
+        app:layout_minHeight="300dp" />
```

Comment:
I think this test case can't be always true.
In fact it failed with my environment (Nexus5x emulator)

In some situations, text1 isn't trying shrink to less than 300dp
(e.g. If the parent's height is 800dp, then text1 and text2 are trying to be expanded to 400dp.
Could you make the test deterministic?
Also the same for the minwidth test.